### PR TITLE
Messaging 5.0.0 updates

### PR DESCRIPTION
**Pull request by swarna04**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -17,13 +17,13 @@ import PackageDescription
...
+    platforms: [.iOS(.v12)],
     products: [
         .library(name: "AEPMessaging", targets: ["AEPMessaging"])
     ],
     dependencies: [
-        .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .upToNextMajor(from: "4.0.0")),
-        .package(url: "https://github.com/adobe/aepsdk-edge-ios.git", .upToNextMajor(from: "4.0.0")),
+        .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .upToNextMajor(from: "5.0.0")),
+        .package(url: "https://github.com/adobe/aepsdk-edge-ios.git", .upToNextMajor(from: "5.0.0")),
         .package(url: "https://github.com/adobe/aepsdk-edgeidentity-ios.git", .upToNextMajor(from: "4.0.0"))
```
Comment (sbenedicadb) [inline Package.swift:27]: should we update edge identity to 5.0.0 too?

File: Podfile [hunk 2]
```diff
@@ -42,59 +42,59 @@ def app_main
...
+    pod 'AEPAnalytics', :git => 'https://github.com/adobe/aepsdk-analytics-ios.git', :branch => 'dev-v5.0.0'
+    pod 'AEPAssurance', :git => 'https://github.com/adobe/aepsdk-assurance-ios.git', :branch => 'dev-v5.0.0'
 end
 
 # ==================
 # TARGET DEFINITIONS
 # ==================
 target 'AEPMessaging' do
-  lib_main
+  lib_dev
```
Comment (sbenedicadb) [inline Podfile:61]: i guess we'll revert this later

**Review by sbenedicadb — APPROVED** [review]
looks good, one thing that needs updated i believe

File: Podfile (hunk 2 above)
Comment (swarna04) [reply Podfile:61]: yup

**Pull request merged**